### PR TITLE
Adjustments to loading indicators in results view for smoother initial load

### PR DESCRIPTION
**Pull request by adamabeshouse**

### CODE REVIEW

File: src/pages/resultsView/ResultsViewPage.tsx [hunk 1]
```diff
@@ -703,10 +704,8 @@ export default class ResultsViewPage extends React.Component<
                                 {// we don't show the result tabs if we don't have valid query
                                 this.showTabs &&
                                     !this.resultsViewPageStore.genesInvalid &&
-                                    !this.resultsViewPageStore.isQueryInvalid &&
-                                    this.resultsViewPageStore
```
Comment (alisman) [inline src/pages/resultsView/ResultsViewPage.tsx:707]: I think @pvannierop added this for a reason.  We talked about moving it deeper but ...

Comment (alisman) [reply src/pages/resultsView/ResultsViewPage.tsx:707]: it means we load all mutation data before we show tabs

Comment (adamabeshouse) [reply src/pages/resultsView/ResultsViewPage.tsx:707]: why do we want to do that? the tabs handle their own loading state/indicators

Comment (alisman) [reply src/pages/resultsView/ResultsViewPage.tsx:707]: @pvannierop lets talk about whether we can put this further down or not.   but @adamabeshouse maybe we just need to add a loader when these conditions aren't true?

Comment (alisman) [reply src/pages/resultsView/ResultsViewPage.tsx:707]: @adamabeshouse i agree.  i know this came up before but i can't remember why it ended up here.

File: src/pages/resultsView/ResultsViewPage.tsx [hunk 2]
```diff
@@ -609,6 +609,13 @@ export default class ResultsViewPage extends React.Component<
                 </div>
             );
         } else {
+            // we don't show the result tabs if we don't have valid query
+            const tabsReady =
```
Comment (alisman) [inline src/pages/resultsView/ResultsViewPage.tsx:613]: i'm gonna merge this right now, but why don't we put this in showTabs?

**Review by alisman — APPROVED** [review]

**Pull request merged**